Source/Modules/xml.cxx

```diff
@@ -146,7 +146,10 @@ class XML:public Language {
 	Xml_print_parmlist(Getattr(obj, k));
       } else if (Cmp(k, "catchlist") == 0) {
 	Xml_print_parmlist(Getattr(obj, k), "catchlist");
-      } else {
+      } else if (Cmp(k, "templateparms") == 0) {
```

Comment:
Maybe unify this with the `if` above, especially if there could be more such cases in the future? I.e. just do

      } else if (Cmp(k, "catchlist") == 0 || Cmp(k, "templateparms") == 0) {
	Xml_print_parmlist(Getattr(obj, k), k);
      } else {

?